content/ko/docs/concepts/workloads/pods/pod-overview.md

```diff
@@ -0,0 +1,107 @@
+
+한 개의 파드 안에서 함께 배치되고 관리되는 다중 컨테이너 그룹은 상대적으로 심화된 사용 예시임에 유의하자. 당신은 컨테이너들이 강하게 결합된 특별한 인스턴스의 경우에만 이 패턴을 사용하는게 좋다. 예를 들어, 공유 볼륨안의 파일을 위한 웹 서버와 같이 동작하는 컨테이너와 원격 소스로 부터 그 파일들을 업데이트 하는 분리된 "사이드카" 컨테이너가 있는 경우 아래 다이어그램의 모습일 것이다.
+
+
+{{< figure src="/images/docs/pod.svg" title="pod diagram" width="50%" >}}
+
+파드는 같은 파드 안에 속한 컨테이너들에게 두가지 공유 리소스를 제공한다: *네트워킹* 과 *저장소*.
+
+#### 네트워킹
+
+각각의 파드는 유일한 IP주소를 할당 받는다. 한 파드 안의 모든 컨테이너는 네트워크 네임스페이스와 IP주소 및 네트워크 포트를 공유한다. *파드 안에 있는* 컨테이너들은  다른 컨테이너들과 `localhost`를 통해서 통신할 수 있다. 특정 파드 안에 있는 컨테이너들이 *파드 밖의* 요소들과 통신하기 위해서는, 네트워크 리소스를 어떻게 쓰고 있는지 공유 받아야 한다(예를 들어 포트 등).
+
+#### 저장소
+
+파드는 공유 저장소 세트인 *볼륨*을 명시할 수 있다. 파드 안의 모든 컨테이너들은 공유 볼륨에 접근할 수 있고, 그 컨테이너들끼리 데이터를 공유하는 것을 허용한다. 또한 볼륨은 컨테이너가 재시작되어야 하는 상황에도 파드 안의 데이터가 영구적으로 유지될 수 있게한다. 쿠버네티스가 어떻게 파드 안의 공유 저장소를 사용하는지 보려면 [볼륨](/docs/concepts/storage/volumes/)를 참고한다.
+
+## 파드와 작업
+
+당신은 직접 쿠버네티스에서 싱글톤 파드이더라도 개별 파드를 만들일이 거의 없을 것이다. 그 이유는 파드가 상대적으로 수명이 짧고 일시적이기 때문이다. 파드가 만들어지면(당신이 직접 만들거나, 컨트롤러에 의해서 간접적으로 만들어지거나), 그것은 당신의 클러스터의 노드에서 동작할 것이다. 파드는 프로세스가 끝나거나, 파드 객체가 삭제되거나, 파드가 리소스의 부족으로 인해 *철거되거나*, 노드에 장애가 생기지 않는 한 노드에 남아있는다. 
+
+{{< note >}}
+파드 안에서 재시작되는 컨테이너와 재시작되는 파드와 혼동해서는 안된다. 파드는 자기 스스로 동작하지 않는다. 하지만 컨테이너 환경은 그것이 삭제될 때 까지 계속 동작한다.
+{{< /note >}}
+
+파드는 스스로 자신을 치료하지 않는다. 만약 파드가 스케쥴링된 노드에 장애가 생기거나, 스케쥴링 동작이 스스로 실패하거나, 파드가 삭제된다; 비슷하게, 파드는 리소스나 노드의 유지 부족으로 인해 살아 남지 못할 것이다.
+쿠버네티스는 *컨트롤러* 라고 하는 고수준의 추상적 개념을 사용하는데, 컨트롤러는 상대적으로 일시적인 파드 인스턴스를 관리하는 작업을 처리한다. 즉, 이것이 파드를 직접적으로 사용할 수 있게 하고 쿠버네티스에서 컨트롤러로 파드를 관리하는 것은 상당히 보편적이다. 쿠버네티스가 어떻게 파드 스케일링과 치료하는지 보려면 [파드와 컨트롤러](#pods-and-controllers) 를 참고한다.
+
+### 파드와 컨트롤러
+
+컨트롤러는 당신을 위해 다중 파드를 생성하고 관리해 주는데, 클러스터 범위 내에서의 레플리케이션 핸들링, 롤아웃 그리고 자가치료 기능 제공을 한다. 예를 들어, 만약 노드가 고장났을 때, 컨트롤러는 아마 자동으로 다른 노드에 고장난 노드에 스케줄링되고 있는 것과 같은 파드로 교체할 것이다.  
+
+한가지 또는 그 이상의 파드를 보유한 컨트롤러의 몇가지 예시:
+
+* [디플로이먼트](/docs/concepts/workloads/controllers/deployment/)
+* [스테이트풀 셋](/docs/concepts/workloads/controllers/statefulset/)
+* [데몬 셋](/docs/concepts/workloads/controllers/daemonset/)
+
+일반적으로, 컨트롤러는 당신이 책임을 지고 제공한 파드 템플릿을 사용한다.
+
+## 파드 
+파드 템플릿은 [레플리케이션 컨트롤러](/docs/concepts/workloads/controllers/replicationcontroller/), [잡](/docs/concepts/jobs/run-to-completion-finite-workloads/), [데몬 셋](/docs/concepts/workloads/controllers/daemonset/)과 같은 다른 객체를 포함하는 파드 명세서이다. 컨트롤러는 파드 템플릿을 실제 파드를 만드는데 사용한다.
+아래 샘플은 메시지를 출력하는 컨테이너를 포함하는 파드에 대한 간단한 매니페스트이다.
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: myapp-pod
+  labels:
+    app: myapp
+spec:
+  containers:
+  - name: myapp-container
+    image: busybox
+    command: ['sh', '-c', 'echo Hello Kubernetes! && sleep 3600']
+```
+
+파드는 모든 레플리카의 현재 원하는 상태를 지정한다라고 하기 보다는, 쿠키 틀과 같다고 보는게 더 좋다. 쿠키가 한 번 잘리면, 그 쿠키는 쿠키 틀과 더이상 관련이 없다. 양자 얽힘이 없는 것이다. 그 이후 템플릿을 변경하거나 새로운 템플릿으로 바꿔도 이미 만들어진 파드들에는 직접적인 영향이 없다. 마찬가지로, 레플리케이션 컨트롤러에 의해 만들어진 파드들은 아마 그 이후 직접 업데이트 될 수 있다. 이것은 모든 컨테이너가 속해있는 파드에서 현재 원하는 상태를 명시하는 것과 의도적으로 대비가 된다. 이러한 접근은 시스템의 의미를 철저히 단순화 시키고 유연성을 증가시킨다.
```

Comment:
> *origin*
>
> Rather than specifying the current desired state of all replicas, pod templates are like cookie cutters. 
> Once a cookie has been cut, the cookie has no relationship to the cutter. 
> There is no "quantum entanglement".
> Subsequent changes to the template or even switching to a new template has no direct effect on the pods already created.
> Similarly, pods created by a replication controller may subsequently be updated directly.
> This is in deliberate contrast to pods, which do specify the current desired state of all containers belonging to the pod.
> This approach radically simplifies system semantics and increases the flexibility of the primitive.

- [ ] **[L99-1]**
`Rather than specifying the current desired state of all replicas, pod templates are like cookie cutters.` 원문의 경우 아래와 같이 번역하는 것이 더 자연스러울 것 같습니다. 

     > 모든 레플리카의 현재 원하는 상태를 지정하는 대신, 파드 템플릿은 쿠키 틀과 같다. 

- [ ] **[L99-2]**
`업데이트 될` -> 업데이트될`. 

- [ ] **[L99-3]**
**[L20-2]** 의 선택에 따라 `Pods` 용어 통일이 필요합니다. 

- [ ] **[L99-4]**
`단순화 시키고` -> `단순화시키고`.